### PR TITLE
Branding-Advertising

### PR DESCRIPTION
**Pull request by SondraE**
## Steps to test/reproduce
/services/development/

### CODE REVIEW

**Comment by SondraE** [comment]
[Add specific services language to our site per trademark counsel's request.](https://trello.com/c/6Zkly3Ds/159-add-specific-services-language-to-our-site-per-trademark-counsels-request)

File: content/services/development.md [hunk 1]
```diff
@@ -75,6 +75,30 @@ summary: |
 
 {% endcall %}
 
+## Design & Strategy
```
Comment (jgerigmeyer) [inline content/services/development.md:78]: We can try this and see. My hunch is that he's looking for specific usage of the terms "branding" and "advertising", and you've successfully removed most of those from the earlier version of the text. 😄

**Review by jgerigmeyer — APPROVED** [review]
@mirisuzanne?

**Comment by SondraE** [comment]
I'd like to add a Development paragraph too. The longer term plan is to separate Design and Development into two separate Services and go into more detail about branding on the Design page, including sample logos and graphics.

File: content/services/development.md (hunk 1 above)
Comment (SondraE) [reply content/services/development.md:78]: Why do we need to add "advertising" if we don't offer that service?

Comment (SondraE) [reply content/services/development.md:78]: We could change the title to "Design & Branding" for now, if needed.

Comment (jgerigmeyer) [reply content/services/development.md:78]: Because the trademark classes cover the following, and I think he's looking for samples to support each one:

- Branding services, namely, consulting, development, management and marketing of brands for businesses and/or individuals
- Advertising services, namely, creating corporate logos for others
- Consulting services in the field of providing online, non-downloadable software and applications
- Internet-based application service provider, namely, hosting, managing, developing, analyzing, and maintaining the code, applications, and software for web sites of others
- Design and development of on-line computer software systems

Comment (mirisuzanne) [reply content/services/development.md:78]: We do offer branding stuff, and I think that's what we intended with our trademark - so we could at least try and highlight that more.

```suggestion
## Design, Branding & Strategy
```

File: content/services/development.md [hunk 2]
```diff
@@ -75,6 +75,30 @@ summary: |
...
+
+**We provide the full range of design services** --
+from user research & strategy
+to branding,
+from user experience & user interface design
+to integrated design systems
+and consulting around marketing & SEO.
+Deliverables include:
+logos & wordmarks,
+brand style guides,
```
Comment (mirisuzanne) [inline content/services/development.md:88]: I found the repeated from/to structure hard to follow. Just an idea for cleaning that up:

```suggestion
**We provide the full range of design & branding services** --
from user research & strategy
to brand and logo design,
user experience & interface design,
integrated design systems,
and consulting around marketing & SEO.
Deliverables include:
logos & wordmarks,
brand style guides,
```

**Review by mirisuzanne — COMMENTED** [review]
I like this as a first step. Maybe good to also start on a more detailed services page? I proposed a few changes for clarity (and a bit more attention to the brand/logo stuff)
